### ISSUE TITLE
o-forms: low contrast for inverted toggle inputs - ID: DAC_Non_Text_Contrast_09

### ISSUE THREAD

**Issue by JakeChampion** [Financial-Times/origami#541]
# Issue
This is from page 124 of the [audit report](https://drive.google.com/file/d/17wq6b0CIX41RA4muBFDzUDxE0-qzHBIH/view)

Issue can be found on: <https://www.ft.com/__origami/service/build/v3/demo?component=o-forms@9.0.1&demo=toggle&system_code=origami-repo-data&brand=master>

The colour contrast of the inversed toggle buttons fails to meet WCAG 2.1 guidelines for non-text contrast. This issue may affect low vision users who may find difficulty in distinguishing the toggle buttons against the background colour. To meet WCAG AA non- text content guidelines the minimum contrast ratio is 3.1 (it is currently 1.5:1 – Inverse 1 and 1.1:1 – Inverse 2).
 
# Proposed Solution:
Ensure the colour contrast of the toggle button meets a minimum of 3.1.
